src/components/HomePage/RecentBlogItem.jsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import BambooStraw from "../../images/BambooStraw";
+
+
+const RecentBlogItem = (title, description, image) => {
+
+  title = "Why buying bamboo straws ?!";
+  description =
+   "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quis netus gravida vitae at dui. Arcu ullamcorper eget lorem nulla sit. Elit varius felis praesent vel. Commodo quis nisi leo tristique in..."; 
+  image = <BambooStraw/>;
+  
+
+
+  return (
+    <div className=" container flex">
+      <div className="static flex-start p-1.5">
```

Comment:
there is no such a thing as `flex-start` in tailwind, also you can't use flex properties without changing the display to flex at first